util/conversion/conversion.go

```diff
@@ -50,3 +62,28 @@ func UnmarshalData(from metav1.Object, to interface{}) (bool, error) {
 	delete(from.GetAnnotations(), DataAnnotation)
 	return true, nil
 }
+
+// GetFuzzer returns a new fuzzer to be used for testing.
+func GetFuzzer(scheme *runtime.Scheme) *fuzz.Fuzzer {
+	return fuzzer.FuzzerFor(
+		fuzzer.MergeFuzzerFuncs(metafuzzer.Funcs),
```

Comment:
Is this actually fuzzing values in our types? iirc, we need to define our own fuzzing methods for custom types similar to this: https://github.com/kubernetes/kubernetes/blob/550fb1bfc3c773febbb0811262fd7a8f7847bb20/pkg/apis/core/fuzzer/fuzzer.go

Reply:
I'd have to track down what gofuzz is doing, but I don't think it's required but rather if you want to customize fuzzing?

Reply:
Circling back, yes, this will fuzz custom structs.

Reply:
Yeah it'll work fine with custom structs. Although if you want to have some special fuzzing logic for some types, you can register more functions